extensions/sql-bindings/src/common/utils.ts

```diff
@@ -162,6 +162,18 @@ export function escapeClosingBrackets(str: string): string {
 	return str.replace(']', ']]');
 }
 
+export function checkBrackets(objectName: string): string {
```

Comment:
This is doing more than just "checking" brackets. Make sure your function names clearly indicate what the function is actually doing (and use doc comments for further details if needed)